src/tutorial/microbe_stage/MicrobeTutorialGUI.cs

```diff
@@ -124,6 +129,26 @@ public bool MicrobeWelcomeVisible
 
             if (value)
             {
+                if (CurrentLifeOrigin == WorldGenerationSettings.LifeOrigin.Vent)
+                {
+                    // Dummy localized string
+                    new LocalizedString("MICROBE_STAGE_INITIAL_VENT");
```

Comment:
Instead of this add a private method like:
```
private static void DummyKeepInitialTextTranslations()
{
    TranslationServer.Translate("MICROBE_STAGE_INITIAL_VENT");
    TranslationServer.Translate(... and so on...
}
```